lms/static/scripts/frontend_apps/common/use-element-should-close.js

```diff
@@ -45,7 +45,7 @@ export default function useElementShouldClose(
    *  Helper to return the underlying node object whether
    *  `closeableEl` is attached to an HTMLNode or Preact component.
    *
-   *  @param {Preact ref} closeableEl
+   *  @param {any} closeableEl
```

Comment:
Do you know what type _would_ be appropriate here?

Reply:
`import("preact").Ref<Element>` I suspect. This file came from the client originally so I'll probably copy across the latest version from there.